docs/spec_draft.md

```diff
@@ -2311,6 +2313,49 @@ More formally, for each element `x`: `imag(x) = is_complex(x) ? x.imag : 0.0`.
 
 [Back to Ops](#index-of-ops)
 
+
+## stablehlo.infeed
+
+### Semantics
+
+Reads `results` from the infeed queue of the device.
+
+Not having a total order among the infeed operations in a program might lead to
+data races or even deadlocks.
+
+The infeed configuration string `infeed_config` includes target-dependent
+metadata.
```

Comment:
Similarly to https://github.com/openxla/stablehlo/pull/580/files#r1037788421, proposed rewording:

```
Reads data from the infeed and produces `results`.

Semantics of `infeed_config` is implementation-defined.

`results` consist of payload values which come first and a token which comes
last. The operation produces a token to reify the side effect of this operation 
as a value that other operations can take a data dependency on.
```